templates/eap-xp1-openjdk11-image-stream.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
@luck3y just to be sure we are on the same page, templates for EAP XP 1.0 and EAP 7.3 will be both developed in the 7.3.x branch but they will be "published" in 2 separate branches `eap73` and `eap-xp1`. Is that correct?

Reply:
Yes, that's the thought anyway -- if we need to maintain very different templates for XP for some reason, this will become difficult though.